front-end/src/Views/ProfilePage.js

```diff
@@ -866,6 +876,53 @@ export default class ProfilePage extends Component {
     )
   };
 
+  getPublications = () => {
+    var userId = getUserId();
+    console.log(userId);
+    axios.get(`${config.API_URL}/api/publications/?owner__id=${userId}`, { headers: { 'Content-Type': 'Application/json', 'Authorization': `Token ${getAccessToken()}` } })
+      .then(res => {
+        let data = res.data;
+        //console.log(data);
+
+        var myPubs = [];
+        for(var i=0; i<10; i++){
+          var myData = [i, data[i].title, data[i].publication_year, data[i].citation_number, data[i].link];
+          myPubs.push(myData);
+        }
+
+        this.setState({publications: myPubs});
+        console.log(this.state.publications);
+      });
+  };

```

Comment:
if a user has no publication, this part will not work

Reply:
thanks for the comment, the bug has fixed and the pr is ready to merge